### ISSUE TITLE
Listview Item Templates Horizontal orientation

### ISSUE THREAD

**Issue by potterdev** [praeclarum/Ooui#141]
Hi,

Thanks a lot, this library is awesome!

I'm working on a project and can't seem to get the orientation of the contents listview items to horizontal. They always stack on top of each other. I tried using either a grid with columns, or a StackLayout with horizontal orientation, but nothing seems to work. Also the background of the viewcell seems to be black for some reason.

Here is an example 
    
    <ListView.ItemTemplate>
      <DataTemplate>
        <ViewCell>
          <StackLayout Orientation="Horizontal">
            <Label Text="{Binding .}" 
                   Style="{DynamicResource ListItemTextStyle}" />
            <Label Text="{Binding .}" 
                   Style="{DynamicResource ListItemDetailTextStyle}"/>
          </StackLayout>
        </ViewCell>
      </DataTemplate>
    </ListView.ItemTemplate>

Both labels are on stacked.

**Comment by praeclarum** [comment]
Yeah, I know! What's up with that?! Sorry for the bug, need to look into it.

**Comment by vignatov** [comment]
Any solution to this?
May be a workaround?

**Comment by vignatov** [comment]
After a little digging and debugging the code, I found that events `BatchCommitted` and `SizeChanged` are not called for elements inside list view item. Not sure they are supposed to be called by Xamarin, it needs investigating. If not then may be the manual propagation of those events to ListView item children is required.

**Comment by sdaves** [comment]
Hey @vignatov did you find a workaround to the issue?
